client/app/resolvers/forum/messages.ts

```diff
@@ -0,0 +1,15 @@
+import { getLocal, postLocal } from "../..";
+
+type ThreadMessagesServerResult = {
+    thread: any;
+    messages: any;
+};
+
+export const resolveMessages = (id: string) =>
+    getLocal<ThreadMessagesServerResult>(`db/thread/${id}`).then((thread) => {
```

Comment:
Если в `then` данные никак не модифицируются, то он и не нужен

```suggestion
    getLocal<ThreadMessagesServerResult>(`db/thread/${id}`)
```

В других резолверах тоже